tests/integration/targets/ec2_metadata_facts/playbooks/test_initial_metadata.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
We should probably add a wait_for_connection entry here.  The failure looks to be an auth failure which is likely the connection occurring before the user-data finishes running.